### ISSUE TITLE
Snapshot created and used only for first disk image in VMs with multiple disk images

### ISSUE THREAD

**Issue by zachbthomas** [sandialabs/sceptre-phenix#145]
When specifying multiple disk images (multiple volumes) for a VM in a topology, the minimega script is incorrectly generated. Only the first disk image is snapshotted. Subsequent disk images do not get a snapshot file, and as a result, in subsequent disk images the original image file is specified in the experiment's minimega file rather than snapshot files for each disk image

![image](https://github.com/sandialabs/sceptre-phenix/assets/42255542/4f2b1bc3-99e1-45a5-bbe8-adb7ab669ba4)
![image](https://github.com/sandialabs/sceptre-phenix/assets/42255542/ba602094-3e66-436e-a93b-71ae575a26ca)

**Comment by activeshadow** [comment]
Thanks @zachbthomas for creating this issue. What exactly are you expecting to see with multiple disk images? A snapshot for all of them so the base image doesn't get modified during an experiment?

**Comment by activeshadow** [comment]
Thanks @zachbthomas for creating this issue. What exactly are you expecting to see with multiple disk images? A snapshot for all of them so the base image doesn't get modified during an experiment?

**Comment by zachbthomas** [comment]
Yeah exactly. Each disk image gets a snapshot, and pressing the “create a
backing image” button in phenix uses each snapshot file and creates a new
.qc2 file for each.

**Comment by bjwrigh** [comment]
Currently the behavior is such that the second disk is always being written to directly whereas the _snapshot file from the first disk is the diff file from the first drive.  In other words, we are not preserving the second disk in any way.

**Comment by activeshadow** [comment]
@bjwrigh I thought minimega created a snapshot disk in /tmp/minimega for all disks connected to a VM. Is that not the case?
